install/deployment_yamls/k8s/meshery-deployment.yaml

```diff
@@ -35,5 +37,12 @@ spec:
           ports:
             - containerPort: 8080
           resources: {}
+        - env:
+            - name: POSTGRES_PASSWORD
```

Comment:
```suggestion
            - name: DATABASE_PASSWORD
```

Reply:
This is a postgres specific environment variable. Postgres expects it to be `POSTGRES_PASSWORD`